ios/brave-ios/Sources/Brave/Frontend/Browser/TabManager.swift

```diff
@@ -1170,23 +1170,6 @@ class TabManager: NSObject {
     configuration.processPool = WKProcessPool()
   }
 
-  static fileprivate func tabsStateArchivePath() -> String {
```

Comment:
This was just unused and I noticed it used the same folder search engines was using so I deleted it